cpu/cortexm_common/Makefile.include

```diff
@@ -3,5 +3,6 @@ INCLUDES += -I$(RIOTCPU)/cortexm_common/include
 INCLUDES += -I$(RIOTCPU)/cortexm_common/include/vendor
 
 USEMODULE += cortexm_common_periph
+include $(RIOTCPU)/cortexm_common/Makefile.dep
```

Comment:
cannot include Makefile.dep from a different makefile

Reply:
I think it didn't work without, but I will check again